exercises/practice/word-count/src/lib.cairo

```diff
@@ -0,0 +1,3 @@
+pub fn word_count(phrase: ByteArray) -> Felt252Dict<u64> {
```

Comment:
Although returning `Felt252Dict` most closely resembles what canonical data requests, the way dictionaries work in Cairo introduce one bigger issue, which is that there is no way to assert that the dictionary contains _only a certain set of keys_ (in this case words). This would make implementations that add more words than expected still valid, and pass the tests, even though they should fail.

To address this, return a `Span` of results with the word and corresponding count. You can use [word-search](https://github.com/exercism/cairo/blob/main/exercises/practice/word-search/tests/word_search.cairo#L28) as inspiration.

Reply:
Alright. I'd look into this.